CONTRIBUTING.md

```diff
@@ -16,6 +16,7 @@ Before you create a new Issue:
 * Check the [Issues](https://github.com/octokit/rest.js/issues) on Github to ensure one doesn't already exist.
 * Clearly describe the issue, including the steps to reproduce the issue.
 * Please include links to the corresponding github documentation.
+* Please provide a minimal code sample to reproduce the issue if possible.
```

Comment:
Could add a 2nd sentence asking for Runkit snippet such as https://runkit.com/gr2m/5aa034f1440b420012a6eebf. It is really _so very helpful_ if people create these